CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 - GPU.js constructor needs to be provided directly to engine configuration. (#355)
+- A **breaking change**: `'` symbol on the input is preserved in the value. (#617)
```

Comment:
It's a bug fix IMO. Of course the changed results are a breaking change, but we should first decide if the results should change at all.

Reply:
It changes the semantic that we agreed upon. For me it is kind of breaking.